pcdsdevices/attenuator.py

```diff
@@ -1155,14 +1268,18 @@ def format_status_info(self, status_info):
             for cpt in self.lightpath_cpts
         ]
 
-        table = '\n'.join(render_ascii_att(cpt_states, start_index=1))
+        table = '\n'.join(render_ascii_att(cpt_states,  start_index=1))
+
         return f"""
 {table}
 Transmission (E={energy} keV): {transmission}
 Transmission for 3rd harmonic (E={energy_3} keV): {transmission_3}
+{self.error_summary.get()}
```

Comment:
I think `.get()` here shouldn't be used inside `format_status_info`, rather you should be getting the value out of the `status_info` dict like all the other elements here. I think adjusting this will likely fix the unit tests.

The formatting of the info and the getting of the info are separated in these status info formatters so that they can be tested independently and so that a failure to get the data can't possibly completely break the status printout.

Reply:
yes, I just went through and loaded the error summary using status_info and the tests don't fail anymore. still need to toy with the formatting when we have multiple lines of error text.